api/src/services/users/users.test.ts

```diff
@@ -174,60 +184,179 @@ describe('user queries', () => {
-      input: { email, name, role, agencyId: scenario.agency.one.id, isActive },
+
+    scenario('updates a user', async (scenario: StandardScenario) => {
+      mockCurrentUser({
+        id: scenario.user.one.id,
+        email: scenario.user.one.email,
+        roles: ['USDR_ADMIN'],
+      })
+      const email = 'String2@gmail.com'
+      const name = 'FDR'
+      const role = 'ORGANIZATION_STAFF'
+      const isActive = false
+      const original = (await user({ id: scenario.user.one.id })) as User
+      const result = await updateUser({
+        id: original.id,
+        input: {
+          email,
+          name,
+          role,
+          agencyId: scenario.agency.one.id,
+          isActive,
+        },
+      })
+
+      expect(result.email).toEqual(email)
+      expect(result.name).toEqual(name)
+      expect(result.role).toEqual(ROLES.ORGANIZATION_STAFF)
+      expect(result.agencyId).toEqual(scenario.agency.one.id)
+      expect(result.isActive).toEqual(false)
     })
 
-    expect(result.email).toEqual(email)
-    expect(result.name).toEqual(name)
-    expect(result.role).toEqual(ROLES.ORGANIZATION_STAFF)
-    expect(result.agencyId).toEqual(scenario.agency.one.id)
-    expect(result.isActive).toEqual(false)
+    scenario('deletes a user', async (scenario: StandardScenario) => {
+      mockCurrentUser({
+        id: scenario.user.one.id,
+        email: scenario.user.one.email,
+        roles: ['USDR_ADMIN'],
+      })
+      const original = (await deleteUser({
+        id: scenario.user.one.id,
+      })) as User
+      const result = await user({ id: original.id })
+
+      expect(result).toEqual(null)
+      expect(mockPassageUser.delete).not.toHaveBeenCalled()
+    })
   })
 
-  scenario('deletes a user', async (scenario: StandardScenario) => {
-    mockCurrentUser({
-      id: scenario.user.one.id,
-      email: scenario.user.one.email,
-      roles: ['USDR_ADMIN'],
+  describe('<Passage Auth>', () => {
+    beforeAll(() => {
+      process.env.AUTH_PROVIDER = 'passage'
+      process.env.PASSAGE_API_KEY = 'fake_api_key'
+      process.env.PASSAGE_APP_ID = 'fake_app_id'
     })
-    const original = (await deleteUser({
-      id: scenario.user.one.id,
-    })) as User
-    const result = await user({ id: original.id })
 
-    expect(result).toEqual(null)
+    scenario('creates a user', async (scenario: StandardScenario) => {
+      const passageUser = { id: 'new-id-1' }
+      mockPassageUser.create.mockReturnValue(passageUser)
+      mockPassageUser.activate.mockReturnValue(passageUser)
+
+      mockCurrentUser({
+        id: scenario.user.one.id,
+        email: scenario.user.one.email,
+        roles: ['USDR_ADMIN'],
+      })
+
+      const result = await createUser({
+        input: {
+          email: 'uniqueemail2@test.com',
+          name: 'String',
+          agencyId: scenario.agency.one.id,
+          role: 'USDR_ADMIN',
+        },
+      })
+
+      expect(result.passageId).toEqual('new-id-1')
```

Comment:
Recommend checking whether the passage-user functions were called
```suggestion
      expect(result.passageId).toEqual('new-id-1')
      expect(mockPassageUser.create).to.haveBeenCalled()
      expect(mockPassageUser.activate).to.haveBeenCalled()
```